modules/cluster-logging-collector-log-forwarding-supported-plugins.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// logging/cluster-logging-external.adoc
+
+[id="cluster-logging-collector-log-forwarding-supported-plugins_{context}"]
+
+= Supported plugins for forwarding logs to third-party systems
+
+[role="_abstract"]
+Red Hat OpenShift Logging version 5.0 tests and supports the following plugins for forwarding logs to third-party systems.
```

Comment:
I think we should not say that we "support the following plugins" since this may come through as us supporting the plugins even outside of our Fluentd.

How about we change this to "Red Hat OpenShift Logging version 5.0 includes the following plugins for forwarding logs to third-party systems."?

Reply:
Updated. Thanks.